readme/api/get_started/plugins.md

```diff
@@ -7,41 +7,57 @@ In this article you will learn the basic steps to build and test a plugin in Jop
+
+The `src/` directory also contains a [manifest.json](https://github.com/laurent22/joplin/blob/dev/readme/api/references/plugin_manifest/) file, which contains the various information about the plugin that was set in the inital generation of the scaffolding, such as its name, homepage URL, etc. You can edit this at any time, but editing it after it has been published may cause users to have to download it again.
+
+## Setup Source Control 
+
+In your plugin directory, run: 
+
+	git init 
 
-The `src/` directory also contains a [manifest.json](https://github.com/laurent22/joplin/blob/dev/readme/api/references/plugin_manifest/) file, which you can edit to set various information about the plugin, such as its name, homepage URL, etc.
+This will setup source control.
+
+
+## Run Joplin in Development Mode
+
+You should test your plugin in Development Mode. Doing so means that Joplin will run using a different profile, so you can experiment with the plugin without risking to accidentally change or delete your data.
 
 ## Building the plugin
 
-The file `src/index.ts` already contain some basic code meant for testing the plugin. In particular it contains a call to [joplin.plugins.register](https://joplinapp.org/api/references/plugin_api/classes/joplinplugins.html), which all plugins should call to register the plugin. And an `onStart()` event handler, which will be executed by Joplin when the plugin starts.
+From the scaffolding, `src/index.ts` now contains the basic code for a Hello World plugin. 
+
+Two things to note:
+1. It contains a call to [joplin.plugins.register](https://joplinapp.org/api/references/plugin_api/classes/joplinplugins.html). All plugins call this to register the plugin in the app.
+2. An `onStart()` event handler method, which is called when the plugin starts.
 
 To try this basic plugin, compile the app by running the following from the root of the project:
 
 	npm run dist
 
-Doing so should compile all the files into the `dist/` directory. This is from here that Joplin will load the plugin.
-
-## Testing the plugin
-
-In order to test the plugin, you might want to run Joplin in [Development Mode](https://github.com/laurent22/joplin/blob/dev/readme/api/references/development_mode.md). Doing so means that Joplin will run using a different profile, so you can experiment with the plugin without risking to accidentally change or delete your data.
+Doing so should compile all the files into the `dist/` directory. This is where Joplin will load the plugin.
 
-Finally, in order to test the plugin, open the Setting screen, then navigate the the **Plugins** section, and add the plugin path in the **Development plugins** text field. For example, if your plugin project path is `/home/user/src/joplin-plugin`, add this in the text field.
+## Install the plugin
+Open Joplin `Settings/Preferences > Plugins` section. Under Advanced Settings, add the plugin path in the `Development plugins text field. 
```

Comment:
I think we standardise Settings/Preferences to just "Configuration" so please could you change to "Configuration > Plugins"?